profiles/anithasubramanian19.json

```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/87901316/202974132-6b2abe8b-01c9-4547-b904-9cca271990b9.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/203156659-64e768dd-ca0b-48e7-a933-3585a5b4df5c.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/203156659-64e768dd-ca0b-48e7-a933-3585a5b4df5c.png",
```